macOS_environment.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
I don't see osx_environment.sh removed.

Reply:
I'd say let's use snake_case for the file name here, as in macos_environment.sh to keep consistency with the rest of build scripts.

Reply:
Fixed, commit incoming.

Reply:
Fixed in a1bc7a32.